### ISSUE TITLE
Chunk documentation down into even smaller chunks

### ISSUE THREAD

**Issue by kriskuiper** [kriskuiper/cmd-digital-playground#60]
## Description
I think if we chunk it down to as small chunks as possible it will be more pleasant to read and understand.

**Comment by deannabosschert** [comment]
Yes, on it! We first have to figure out/refactor the most logical places to serve [x] info to [x] stakeholder/user.

**Comment by deannabosschert** [comment]
Wait for design review tomorrow.
